quickfixj-core/src/main/java/quickfix/EditableDataDictionaryProvider.java

```diff
@@ -0,0 +1,13 @@
+package quickfix;
+
+
+import quickfix.field.ApplVerID;
+
+/**
+ * Editable dictionary provider used by the DefaultSessionFactory
```

Comment:
I'm not sure this Javadoc is particularly useful and if the DefaultSessionFactory gets re-factored nobody will remember to update this comment (at which point it will be actively unhelpful).